content/3_Silver/Graph_Traversal.mdx

```diff
@@ -1082,6 +1082,49 @@ public class BuildingTeams {
 ```
 
 </JavaSection>
+<PySection>
+
+<Warning>
+You have to submit with CPython (not PyPy3) to avoid exceeding the time limit.
+</Warning>
+
+```py
+import sys
+
+input = sys.stdin.readline
```

Comment:
is just using `input()` too slow

Reply:
Yep, just tested it

https://cses.fi/problemset/result/11126765/ fails
https://cses.fi/problemset/result/11126767/ passes